### PR TITLE
[TASK] Translate to EN-GB or EN-US

### PR DESCRIPTION
**Pull request by vishnu-web-vision**
T3C-82

### CODE REVIEW

File: ext_conf_template.txt [hunk 1]
```diff
@@ -8,3 +8,5 @@ deeplFormality = default
 googleapiKey =
 # cat=Settings/O; type=string; label= Google Translate API Url (Non freemode)
 googleapiUrl = https://translation.googleapis.com/language/translate/v2
+# cat=Settings/O; type=string; label= Traget Language (ISO 369-1) (Define translation target language explicitly)
```
Comment (calien666) [inline ext_conf_template.txt:11]:
```suggestion
# cat=Settings/O; type=string; label= Traget Language (ISO 639-1) (Define translation target language explicitly)
```

File: Classes/Hooks/TranslateHook.php [hunk 2]
```diff
@@ -113,6 +113,24 @@ public function processTranslateTo_copyAction(string &$content, array $languageR
...
+                        } else {
+                            $targetLanguageIso = $targetLanguage->getTwoLetterIsoCode();
+                        }
+
+                        //check for translation target language defined explicitly through extension configuration
+                        if (isset($GLOBALS['TYPO3_CONF_VARS']['EXTENSIONS']['wv_deepltranslate']['translationTargetLanguage'])) {
+                            $translationTargetLanguage = $GLOBALS['TYPO3_CONF_VARS']['EXTENSIONS']['wv_deepltranslate']['translationTargetLanguage'];
+                            if ($translationTargetLanguage != '') {
+                                $targetLanguageIso = $translationTargetLanguage;
+                            }
```
Comment (calien666) [inline Classes/Hooks/TranslateHook.php:131]: As I see, there is no check, if the given specific string only contains the ISO-639-1 language key or if there is more input in defined string. This should be checked, before giving the input to variable.

**Comment by calien666** [comment]
#92

**Comment by NarkNiro** [comment]
With the PR #107 there is now UK translation support. Therefore this PR can be closed.

**Pull request closed without merging**